download/download.go

```diff
@@ -17,23 +26,155 @@ Download: Downloads files from the Sensitive Data Archive (SDA). If a directory
           download the urls_list.txt file, and then download all files in this
           list. If file urls are given, they will be downloaded as-is.
 `
+
+// ArgHelp is the suffix text that will be displayed after the argument list in
+// the module help
 var ArgHelp = `
   [urls]
         all flagless arguments will be used as download urls.`
 
+// Args is a flagset that needs to be exported so that it can be written to the
+// main program help
 var Args = flag.NewFlagSet("download", flag.ExitOnError)
 
-// Main download function
-func Download(args []string) {
+// Gets the file name for a URL, using regex
+func getFileNameFromURL(file string) (fileName string, err error) {
```

Comment:
```suggestion
func createFilePathFromURL(file string) (fileName string, err error) {
```
Changing the name to better match what is going on.